tests/integration/test_integration.py

```diff
@@ -1835,43 +1842,42 @@ def test_ks_2sample_random(engine, random_normal_table, configuration):
 @pytest.mark.parametrize(
     "configuration",
     [
-        ("value_0_1", "value_0_1"),
-        ("value_0_1", "value_02_1"),
-        ("value_0_1", "value_1_1"),
+        ("value_0_1", "value_0_1", 0.0, 1.0),
+        ("value_0_1", "value_005_1", 0.0294, 0.00035221594346540835),
+        ("value_0_1", "value_02_1", 0.0829, 2.6408848561586672e-30),
+        ("value_0_1", "value_1_1", 0.3924, 0.0),
     ],
 )
 def test_ks_2sample_implementation(engine, random_normal_table, configuration):
-    col_1, col_2 = configuration
-    req = requirements.BetweenRequirement.from_tables(
-        *random_normal_table, *random_normal_table
-    )
-    req.add_ks_2sample_constraint(column1=col_1, column2=col_2)
-    constraint = req[0]
-
-    # calculate the 2-sample ks test with db access manually
-    # note: this is also what req[0].test() would do
-    # get query selections and column names for target columns
-    selection1 = str(constraint.ref.data_source.get_clause(engine))
-    column1 = constraint.ref.get_column(engine)
-    selection2 = str(constraint.ref2.data_source.get_clause(engine))
-    column2 = constraint.ref2.get_column(engine)
-
-    # retrieve test statistic d, as well as sample sizes m and n
-    d_statistic, m, n = db_access.get_ks_2sample(
-        engine, table1=(selection1, column1), table2=(selection2, column2)
-    )
+    col_1, col_2, expected_d, expected_p = configuration
+    # different order because schema is optional
```

Comment:
```suggestion
    # different order between values from fixture and `TableDataSource` arguments
```

I'm not sure it's needed to comment on this, but it's not wrong doing it.

Reply:
Yeah you're right, it's not that essential. Will be removed 👍